src/olympia/hero/models.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.db import models
+
+from olympia.amo.models import ModelBase
+from olympia.discovery.models import DiscoveryItem
+
+
+GRADIENT_START_COLOR = '#20123A'
+
+
+class PrimaryHero(ModelBase):
+    image = models.CharField(max_length=255)
+    background_color = models.CharField(max_length=7)
```

Comment:
Do you think it makes sense to keep this as `background_color` when it's being used for the gradient end colour? Perhaps a more descriptive name like `gradient_end_color` would be better?

Reply:
https://github.com/mozilla/addons-server/pull/11858/commits/73904864a04a87e4a736750c627cea266dafa2c4